models/analytics/daily_finances.sql

```diff
@@ -0,0 +1,49 @@
+WITH orders as (
+    SELECT 
+        order_date,
+        sum(clean_shipping_cost) daily_shipping_cost,
+        SUM(total_price) as daily_order_amt
+    FROM {{ ref('int_orders_withprice') }} 
+    GROUP BY 1
+    )
+
+, refund as (
+    SELECT 
+        DATE(returned_at) as returned_date,
+        SUM(CASE WHEN is_refunded = 'yes' THEN total_price ELSE 0 END) as daily_refund,
+    FROM {{ ref('base_googledrive__returns') }}  r
+    JOIN {{ ref('int_orders_withprice') }}  o 
+    ON r.order_id = o.order_id
+    GROUP BY 1
+    )
+
+, expense as (
```

Comment:
100% sure that the date variable is consecutive without any missing values

Reply:
switched the reference table from orders -> expense table